frontend/packages/dev-console/src/components/topology/filters/TopologyFilterBar.tsx

```diff
@@ -44,9 +50,11 @@ const TopologyFilterBar: React.FC<TopologyFilterBarProps> = ({
   onDisplayFiltersChange,
   onSearchChange,
   visualization,
+  consoleLinks,
+  namespace,
 }) => {
   const [searchQuery, setSearchQuery] = React.useState<string>('');
-
+  const kialiLink = getKialiLink(consoleLinks, namespace);
```

Comment:
Not sure if metadata name with `kiali-namespace-{{ namespace }}` pattern would be used for multiple touch points in future enhancements in Kiali. So it might be better to enhance `getKialiLink` utility to fetch link based on location `NamespaceDashboard` & namespace.

Reply:
Updated the `getKialiLink` util.